plugins/git/git.plugin.zsh

```diff
@@ -255,3 +255,21 @@ alias glum='git pull upstream master'
 
 alias gwch='git whatchanged -p --abbrev-commit --pretty=medium'
 alias gwip='git add -A; git rm $(git ls-files --deleted) 2> /dev/null; git commit --no-verify --no-gpg-sign -m "--wip-- [skip ci]"'
+
+function grename() {
+  if [ -z "$1" ]
+  then
+    echo "use command like 'grename old_branch new_branch'"
```

Comment:
The message here could be better, i.e. follow common conventions. This is better:
```zsh
echo "Usage: $0 old_branch new_branch"
```
Also, use `return 1` to indicate a failure to run the command.

Reply:
Thank you for review. I have done some changes please let me know if there is anything i would need to do